cmd/packageExport.go

```diff
@@ -75,6 +75,7 @@ func newPackageExportCommand() *packageExportCommand {
 
 	cmd.Flags().BoolVarP(&template, "template", "t", false, "Export a package config template")
```

Comment:
Thinking about renaming the `template` flag to `example` or something.. not sure I'm happy with that name. Haven't used and looked at this piece of code for quite some time and it took me couple of minutes what was meant with `template`. 

What's your opinion on this?

Reply:
I agree this way it'll be easier to understand… I renamed the flag and the generated file.